fairroot/base/sim/FairModule.h

```diff
@@ -143,7 +143,10 @@ class FairModule : public TNamed
     static thread_local std::vector<FairVolume*> fAllSensitiveVolumes;   //!
 
     TString fMotherVolumeName{""};   //!
-    FairVolume* getFairVolume(FairGeoNode* fNode);
+    /**
+     * \deprecated Not used anywhere, will be removed in v20
```

Comment:
Hm, are we sure we can hold this promise? How about being a bit less specific via "will be removed in a future release" or similar?

Second thought, not sure, if `chore` and deprecation fit. What's your definition for chore?

Reply:
I don't know when / how we can hold those promisses. I just wanted to schedule it somehow.  (Otherwise we will keep it for the next 24 years like `TList(TObject*)` :wink: .)

Using `\deprecated` puts it at least on the deprecation list in Doxygen which we could work on and actually remove things.

Maybe we should decide on our deprecation strategy at some point?

Reply:
My defintion of "chore" is mostly like "clean things up, or prepare to clean up".

That said: I am totally open to alternative tags for deprecations?

Reply:
how about `deprecate(base): FairModule::getFairVolume`?

Reply:
I would vote for the `deprecate(base): FairModule::getFairVolume` comment.

Reply:
done.

looks good to me!